components/Panel/MenuPanel/MenuPanel.tsx

```diff
@@ -225,6 +240,18 @@ const Address = styled.p`
   font: var(--text-sm);
   margin-top: 10px;
   margin-bottom: 15px;
+
+  @media ${mobileAndUnder} {
+    display: none;
+  }
+`;
+
+const TruncatedAddress = styled(Address)`
```

Comment:
Since checking the screen width with the `useWindowSize` hook only works once the page is mounted, I prefer to use this pattern where possible. we have two components and we show / hide them on mobile with media queries and css (doesn't need the page to mount to work).